locales/en.json

```diff
@@ -90,8 +90,8 @@
     "state": {
       "open": "Place open during lockdown",
       "open_adapted": "Place open during lockdown, with adapted hours",
-      "partial": "Place potentially open during lockdown (to be checked)",
-      "unknown": "Place with legal right to open during lockdown, but is it really open or closed ? Let us know",
+      "partial": "Place belongs to a brand whose places are partially open during lockdown. Is it open or closed? Let us know.",
+      "unknown": "Place may be allowed to open during lockdown. Is it open or closed? Let us know.",
       "closed": "Place closed during lockdown"
```

Comment:
By the way, do we really have to mention "lockdown" here?

* open = Place still open
* open_adapted = Place still open with adapted hours
* partial = Place belongs to a brand whose places are partially still open. Is it open or closed? Let us know.
* unknown = Place may be allowed to open. Is it open or closed? Let us know.
* closed = Place temporarily closed

Reply:
I think we don't need to mention lockdown, I find this version even better.

Reply:
See [Florian's answer](https://github.com/osmontrouge/caresteouvert/pull/329#issuecomment-612933304) below.